Documentation corrected (#15944)
kubernetes/website: content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

Add the ConfigMap name under the `volumes` section of the Pod specification.
This adds the ConfigMap data to the directory specified as `volumeMounts.mountPath` (in this case, `/etc/config`).

> The `command` section lists directory files with names that match the keys in ConfigMap.

Reviewer: nit:
```suggestion
The `command` section lists directory files with names that match the keys in the ConfigMap.
```

Reviewer: (although if merged as-is, this is already much better than what's in place).

> The `command` section lists the contents of the directory which got populated with the files having names as same as the keys in configMap.

Reviewer 2: suggest a rewording to make the linkage clearer between the configmap items and what the command does:

"The `command` field of the ContainerSpec describes the command to be executed inside the container. This command simply lists the files in the `/etc/config` directory. The `/etc/config` directory is a mountpoint that refers to a Volume with a name of "config-volume". The Volume's `configMap` field refers to the ConfigMap with a name of "special-config". This ensures that a file for each item in the "special-config" ConfigMap will be written to the `/etc/config` directory."

Author: Indeed this is a very good explanation. But i have never seen such detailing in k8s documentation.  Either such detailed instructions are part of their subsequent topics or there is an assumption that reader already knows about it. Most of the explanation here explains the use of unix `ls` command. Then `command` field which i am sure is basic and has explanation in other documents. After that `volumes` which definitely has plethora of literature. Let me know your thoughts.

> The `command` section lists the contents of the directory which got populated with the files having names as same as the keys in configMap.

Reviewer 3: Grammar fix:
```suggestion
The `command` section lists directory files with names that match the keys in ConfigMap.
```
